src/main/webapp/individuals.jsp

```diff
@@ -724,14 +724,21 @@ if (sharky.getNames() != null) {
         if (newKey===oldKey && newVal===oldVal) return;
 
         var indID = "<%=id%>";
+        var userId = 'null';
```

Comment:
I could see assignment of userId to the string null getting kinda messy. Is there another way to do this?

Reply:
Maybe just '' so that the UI doesn't display, 'null'?